src/leo-client-app/src/components/SatellitesOfInterest.tsx

```diff
@@ -63,6 +85,19 @@ const SatellitesOfInterest = ({ userId }: Props) => {
     fetchData();
   }, []);
 
+  const addSatellite = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    console.log('Satellite Name:', satelliteName, 'NORAD ID:', noradId);
```

Comment:
remove

Reply:
im going to make this change on the other pr because it pertains to the add_satellite pr